src/docs/development/accessibility-and-localization/internationalization.md

```diff
@@ -73,9 +73,9 @@ MaterialApp(
    GlobalWidgetsLocalizations.delegate,
  ],
  supportedLocales: [
-    const Locale('en'), // English
-    const Locale('he'), // Hebrew
-    const Locale('zh'), // Chinese
+    const Locale.fromSubtags(languageCode: 'en'), // English
```

Comment:
It would be simpler to leave English and Hebrew as just simple `Locale` constructor calls, since the advanced Locale features aren't needed.